### ISSUE TITLE
Modification for input vector of real values, not binary.

### ISSUE THREAD

**Issue by snurkabill** [tjake/rbm-dbn-mnist#1]
Hi! 

I would like to ask you, before I will study hard your code, if there is support for Gaussian-Bernoulli RBM. http://goo.gl/NBsyRZ part 1.4.3 ... I am newbie with RBM and I don't understand them as I need to. I have to do some project in school and your code is great source of informations. But I am not sure about real input vector.

**Comment by tjake** [comment]
Yes it does there is a GaussianLayer class that does this.  The minst example is pure binary though.

**Comment by nouramoubayed** [comment]
can you please point to the GaussianLayer class ?

**Comment by tjake** [comment]
https://github.com/tjake/rbm-dbn-mnist/blob/master/src/main/java/com/github/tjake/rbm/GaussianLayer.java

**Comment by sinaahmadi** [comment]
Hi Tjake,
Glad to find your project and to read your article entitled 'Resurgence in Neural Networks'.
I'm having the same problem as mentioned above. I'm gonna use your codes for a real-valued dataset.  As you have replied, the GaussianLayer class is supposed to do so, but in fact I can't find where and how to get use of it. I have already studied about RBM and its formulae and the differences between Gaussian/binary implementations. Would you please tell me which functions you have added(or modified) to convert your binary RBM into a Gaussian? 
Thanks in advance.
